migration_job_stream.go

```diff
@@ -39,6 +41,13 @@ type streamMigrationJob struct {
 }
 
 func (j *streamMigrationJob) Run() {
+	grip.Info(message.Fields{
+		"message":   "starting migration",
+		"migration": j.Definition.Migration,
+		"operation": "stream",
```

Comment:
add operation types to simple and manual too?

Reply:
done